File: JavaFundamentals/src/BasicSyntaxConditionalStatementsAndLoops/Lab/MonthPrinter.java
package BasicSyntaxConditionalStatementsAndLoops.Lab;

import java.util.Scanner;

public class MonthPrinter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int numberOfMonth = Integer.parseInt(scanner.nextLine());
        switch (numberOfMonth) {
            case 1 -> System.out.println("January");
            case 2 -> System.out.println("February");
            case 3 -> System.out.println("March");
            case 4 -> System.out.println("April");
            case 5 -> System.out.println("May");
            case 6 -> System.out.println("June");
            case 7 -> System.out.println("July");
            case 8 -> System.out.println("August");
            case 9 -> System.out.println("September");
            case 10 -> System.out.println("October");
            case 11 -> System.out.println("November");
            case 12 -> System.out.println("December");
            default -> System.out.println("Error!");
        }
    }
}

File: JS Fundamentals/Basic Syntax, Conditional Statements and Loops - Lab/month_printer.js
function printMonth(numb){
    if (numb == 1){
        console.log('January');
    } else if (numb == 2){
        console.log('February');
    } else if (numb == 3){
        console.log('March');
    } else if (numb == 4){
        console.log('April');
    } else if (numb == 5){
        console.log('May');
    } else if (numb == 6){
        console.log('June');
    } else if (numb == 7){
        console.log('July');
    } else if (numb == 8){
        console.log('August');
    } else if (numb == 9){
        console.log('September');
    } else if (numb == 10){
        console.log('October');
    } else if (numb == 11){
        console.log('November');
    } else if (numb == 12){
        console.log('December')
    } else{
        console.log('Error!')
    }
}

printMonth(2);
printMonth(13);